test/api/dependency/api_test.go

```diff
@@ -101,3 +103,106 @@ func TestDependencyList(t *testing.T) {
 		assert.Must(t, Application.Delete(dependency.To.ID))
 	}
 }
+
+func TestReverseDependency(t *testing.T) {
+	for _, r := range ReverseSamples {
+		// Create Applications.
+		assert.Must(t, Application.Create(&r.Application1))
+		assert.Must(t, Application.Create(&r.Application2))
+		assert.Must(t, Application.Create(&r.Application3))
+
+		firstDependency := api.Dependency{
+			From: api.Ref{
+				ID: r.Application1.ID,
+			},
+			To: api.Ref{
+				ID: r.Application2.ID,
+			},
+		}
+		err := Dependency.Create(&firstDependency)
+		if err != nil {
+			t.Errorf(err.Error())
+		}
+
+		secondDependency := api.Dependency{
+			From: api.Ref{
+				ID: r.Application2.ID,
+			},
+			To: api.Ref{
+				ID: r.Application3.ID,
+			},
+		}
+		err = Dependency.Create(&secondDependency)
```

Comment:
Looks OK, but might be easier use ```assert.Should(t,Dependency.Create(&secondDependency))``` instead of condition below (when the API call should not fail).

Reply:
Actually did add the assert.Should condition but tried to maintain uniformity with the failing dependencies, so added this condition